extensions/Worker.Extensions.Storage/Queues/QueueEventTriggerAttribute.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using Microsoft.Azure.Functions.Worker.Extensions.Abstractions;
+
+namespace Microsoft.Azure.Functions.Worker.Extensions.Storage
+{
+    public sealed class QueueEventTriggerAttribute : TriggerBindingAttribute
```

Comment:
I never liked the name `QueueTrigger`. ServiceBus has queues. And those can act as triggers. Perhaps `StorageQueueEventTriggerAttribute`? That way, the code is more intuitive:

```c#
[FunctionName("MyFunc")]
public static void QueueTrigger([StorageQueueTrigger("myqueue")] string item, ILogger log)
{
  log.LogInformation($"C# function processed: {item}");
}
```

Reply:
Azure Storage sdk calls it `Queue` for all their types ([Example](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/storage/Azure.Storage.Queues/src/QueueClient.cs)). I think there's value in keeping parity between the terminology used by the Storage sdk.

And these `Queue` types would be part of the Storage extension package which would be different from the service bus extension package. I think you can also make the same argument for another Storage service that may have queue trigger support for Azure Functions. :)

I've tried to keep these types same as our WebJobs types except some semantical difference to tell them apart easily (Example - Trigger vs EventTrigger). That way it's easy for us to generate the metadata and keep it same across languages. So, unless there's a compelling argument, I would vote to keep the "WebJobs" types and these types consistent (with added convention to tell them apart),